io/io/inc/TFile.h

```diff
@@ -273,7 +273,11 @@ class TFile : public TDirectoryFile {
       kWriteError    = BIT(14),
       kBinaryFile    = BIT(15),
       kRedirected    = BIT(16),
-      kReproducible  = BIT(17)
+      kReproducible  = BIT(17),
+      // Set this bit to block an unwanted automatic call to TTree::ChangeFile
+      // e.g. during TFileMerger operations (bypassing any set MaxTreeSize)
+      // This is a transient bit and has no relevance for I/O.
+      kCannotChange  = BIT(18)
```

Comment:
In this context, this can be misunderstood has saying that the file is "const" and its content "can not change" (i.e. read only I guess).
```suggestion
      kCancelTTreeChangeRequest  = BIT(18)
```
seems better albeit too long.

Reply:
what about kCannotSplit ?

Reply:
This could semantically clash with:
```
TStreamerElement::CannotSplit and TClass::CanSplit
```
I.e the splitting done by `TTree` and `RNTuple`.

Reply:
mmm and what about

kBypassMaxTreeSize ?